data/prices_cryptocompare.txt

```diff
@@ -0,0 +1,1441 @@
+1647.66
```

Comment:
data folder should be in .gitignore

Reply:
if someone runs the code in price-fetcher.py a few days from now, it won't work and they won't have a price data, which is why I didn't put the folder in .gitignore. CryptoCompare can provide minute-level prices for the past 7 days only

Reply:
OK
but for large file output you use https://git-lfs.com/ or just store data somewhere else
github is for storing code not data